examples/cookbook/app/network-requests/__tests__/PhoneBook.test.tsx

```diff
@@ -0,0 +1,119 @@
+import { render, screen, waitForElementToBeRemoved } from '@testing-library/react-native';
+import React from 'react';
+import axios from 'axios';
+import PhoneBook from '../PhoneBook';
+import { User } from '../types';
+
+jest.mock('axios');
+
+jest.setTimeout(10000);
```

Comment:
When this test suit runs in CI, the default 5s is not sufficient.
Locally it's a matter of 0.91 s.

Any idea what might be causing this on CI? @mdjastrzebski?

Reply:
According to my measuring:

```
renders an empty task list ran in 6612ms
PhoneBook fetches contacts successfully and renders in list ran in 7190ms
Top 8 slowest examples (14.071 seconds, 100.0% of total time):
  PhoneBook fetches contacts successfully and renders in list
    7.19 seconds ./app/network-requests/__tests__/PhoneBook.test.tsx
  renders an empty task list
    6.612 seconds ./app/state-management/jotai/__tests__/TaskList.test.tsx
  renders WelcomeScreen in light theme
    0.078 seconds ./app/custom-render/__tests__/index.test.tsx
  PhoneBook fetches favorites successfully and renders all users avatars
    0.0[5](https://github.com/callstack/react-native-testing-library/actions/runs/10590927775/job/29347478417?pr=1655#step:6:6)6 seconds ./app/network-requests/__tests__/PhoneBook.test.tsx
  PhoneBook fails to fetch contacts and renders error message
    0.052 seconds ./app/network-requests/__tests__/PhoneBook.test.tsx
  PhoneBook fails to fetch favorites and renders error message
    0.052 seconds ./app/network-requests/__tests__/PhoneBook.test.tsx
  renders a to do list with 1 items initially, and adds a new item
    0.02[6](https://github.com/callstack/react-native-testing-library/actions/runs/10590927775/job/29347478417?pr=1655#step:6:7) seconds ./app/state-management/jotai/__tests__/TaskList.test.tsx
  renders WelcomeScreen in dark theme
    0.005 seconds ./app/custom-render/__tests__/index.test.tsx
```

Reply:
It seems like the spec. tests "renders an empty task list" and "PhoneBook fetches contacts successfully and renders in list" are the slowest as they're the 1st tests in the file 🤷🏻